pupil-spa/src/app/submission-failed/submission-failed.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <header class="page-header">
+      <h1 class="heading-xlarge">Unable to connect</h1>
+    </header>
+    <div class="panel panel-border-wide">
+      Please contact the helpline on 0345 278 8080 for advice on how to upload the check.
```

Comment:
Should we move telephone numbers into the config?

Reply:
@jon-shipley good point

Reply:
@jon-shipley updated